.github/workflows/rds-plan.yml

```diff
@@ -0,0 +1,35 @@
+name: rds plan terraform
```

Comment:
This name should be "api-rds plan terraform" and the file should be named api-rds-plan.yml